packages/core-data/src/actions.js

```diff
@@ -607,8 +603,8 @@ export const saveEntityRecord = (
  *                         `saveEntityRecord`, `saveEditedEntityRecord`, and
  *                         `deleteEntityRecord`.
  *
- * @return {Promise} A promise that resolves to an array containing the return
- *                   values of each function given in `requests`.
+ * @return {(thinkArgs: Object) => Promise} A promise that resolves to an array containing the return
```

Comment:
Why `thinkArgs`?

Reply:
Looking at the original PR, I think this was meant to say `thunks`
```suggestion
 * @return {(thunkArgs: Object) => Promise} A promise that resolves to an array containing the return
```

Reply:
Thanks for the update; you were spot on - thunkArgs - We could have made a better type here but mostly I want to get things set so we don’t have errors once we turn on TypeScript